cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -450,19 +450,34 @@ func findTestingNodeWithDevicesInTwoNUMANodes(sriovCapableNodes *sriovcluster.En
 
 		numa0Devices := findDevicesOnNUMANode(testingNode, sriovDevices, "0")
 		if len(numa0Devices) == 0 {
-			failureMessage += fmt.Sprintf("node [%s] has no NUMA0 devices, ", nodeName)
+			failureMessage += fmt.Sprintf("node [%s] has no NUMA0 devices\n%s\n", nodeName, dumpLspciForEthernetDevices(testingNode))
 			continue
 		}
 
 		numa1Devices := findDevicesOnNUMANode(testingNode, sriovDevices, "1")
 		if len(numa1Devices) == 0 {
-			failureMessage += fmt.Sprintf("node [%s] has no NUMA1 devices, ", nodeName)
+			failureMessage += fmt.Sprintf("node [%s] has no NUMA1 devices\n%s\n", nodeName, dumpLspciForEthernetDevices(testingNode))
 			continue
 		}
 
 		return testingNode, numa0Devices, numa1Devices
 	}
 
-	Fail("Can't find a suitable node for testing: " + failureMessage)
+	// Invoking ginkgo.Fail directly does not trigger the FailHandler, which generates k8sreporter archives.
+	Expect(false).To(BeTrue(), "Can't find a suitable node for testing: "+failureMessage)
 	return nil, nil, nil
 }
+
+func dumpLspciForEthernetDevices(node *corev1.Node) string {
+	// [0200] is the class for Ethernet controllers
```

Comment:
I think this comment is not relevant anymore

Reply:
correct! fixed